### PR TITLE
Highlight words

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

**Comment by erezh31** [comment]
@meirwah plz review when you have time

File: Scripts/script-HighlightWords.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+tags: []
+comment: Highlight words inside a given text.
+args:
+- name: text
+  required: true
+  description: The input text to hightlight the words in.
+- name: terms
+  required: true
+  description: Terms to hightlight in the text. Can be words or sentances (that does
+    not includes comma). Notice taht if you used a sentances and word, the word should
```
Comment (meirwah) [inline Scripts/script-HighlightWords.yml]: typo in  taht

Comment (meirwah) [reply Scripts/script-HighlightWords.yml]: should -> shouldn't right?

**Comment by erezh31** [comment]
@meirwah comments fixed. plz approve

**Comment by meirwah** [comment]
build fails @erezh31

**Pull request merged**